SubmitReview/action.yml

```
@@ -0,0 +1,12 @@
+name: 'Move card back after submitting review'
```

Foreshadowing the next ticket
```suggestion
name: 'Move card after submitting a review'
```